src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -51,7 +51,10 @@ export class BrowsingContextImpl {
   #eventManager: IEventManager;
   readonly #children: BrowsingContextImpl[] = [];
   #scriptEvaluator: ScriptEvaluator;
-  #executionContext: number | null = null;
+  readonly #sandboxToExecutionContextIdMap: Map<
+    string | null,
```

Comment:
What does `null` mean in this context? We should probably add a doc comment here.

Reply:
Added comments describing. I use `null` for default execution context.